pkg/plugin/plugin.go

```diff
@@ -190,6 +190,7 @@ func SetupPluginEnv(settings helm_env.EnvSettings,
 
 		"TILLER_HOST":      settings.TillerHost,
 		"TILLER_NAMESPACE": settings.TillerNamespace,
+		"TILLER_PORT":      "44134",
```

Comment:
can you change this to use `settings.TillerPort`? That way the port number changes if someone sets --tiller-port or $TILLER_PORT.

Reply:
Yeah I can't remember why I didn't change this before... :confused: